insights/parsers/pam.py

```diff
@@ -108,8 +108,8 @@ class PamSSHD(PamDConf):
 """
 
 from collections import namedtuple
-from .. import Parser, get_active_lines, parser
-from ..parsers import unsplit_lines
+from insights import Parser, get_active_lines, parser
+from insights.parsers import unsplit_lines, keyword_search
```

Comment:
These links should be left as relative to be consistent with the code base.  However, the second line could be simplified by:
```
from . import unsplit_lines, keyword_search
```